stdlib/public/core/Integers.swift.gyb

```diff
@@ -77,56 +77,912 @@ infix operator &>>= : AssignmentPrecedence
+  ///
+  ///     let x = 15
+  ///     print(x.remainder(dividingBy: 7))
+  ///     // Prints "1"
+  ///     print(x % 7)
+  ///     // Prints "1"
+  ///
+  /// - Parameter rhs: The value to divide this value by. `rhs` must not be
+  ///   zero.
+  /// - Returns: The remainder of dividing this value by `rhs`.
+  ///
+  /// - SeeAlso: `formRemainder(dividingBy:)`
+""",
+
+        '&': """\
+  /// Returns the result of performing a bitwise AND operation on this value
+  /// and the given value.
+  ///
+  /// A bitwise AND operation results in a value that has each bit set to `1`
+  /// where *both* of its arguments have that bit set to `1`. For example:
+  ///
+  ///     let x: UInt8 = 5          // 0b00000101
+  ///     let y: UInt8 = 14         // 0b00001110
+  ///     let z = x.bitwiseAnd(y)   // 0b00000100
+  ///
+  /// This method serves as the basis for the bitwise AND operator (`&`).
+  ///
+  /// - Parameter rhs: The value to perform the bitwise AND with this value.
+  /// - Returns: A new value with each bit set to `1` where this value and
+  ///   `rhs` both have bits set to `1`.
+""",
+        '|': """\
+  /// Returns the result of performing a bitwise OR operation on this value and
+  /// the given value.
+  ///
+  /// A bitwise OR operation results in a value that has each bit set to `1`
+  /// where *one or both* of its arguments have that bit set to `1`. For
+  /// example:
+  ///
+  ///     let x: UInt8 = 5          // 0b00000101
+  ///     let y: UInt8 = 14         // 0b00001110
+  ///     let z = x.bitwiseOr(y)    // 0b00001111
+  ///
+  /// This method serves as the basis for the bitwise OR operator (`|`).
+  ///
+  /// - Parameter rhs: The value to perform the bitwise OR with this value.
+  /// - Returns: A new value with each bit set to `1` where this value, `rhs`,
+  ///   or both have bits set to `1`.
+""",
+        '^': """\
+  /// Returns the result of performing a bitwise XOR operation on this value
+  /// and the given value.
+  ///
+  /// A bitwise XOR operation, also known as an exclusive OR operation, results
+  /// in a value that has each bit set to `1` where *one or the other but not
+  /// both* of its arguments had that bit set to `1`. For example:
+  ///
+  ///     let x: UInt8 = 5          // 0b00000101
+  ///     let y: UInt8 = 14         // 0b00001110
+  ///     let z = x.bitwiseXor(y)   // 0b00001011
+  ///
+  /// This method serves as the basis for the bitwise XOR operator (`^`).
+  ///
+  /// - Parameter rhs: The value to perform the bitwise XOR with this value.
+  /// - Returns: A new value with each bit set to `1` where either this value
+  ///   or `rhs`, but not both, have bits set to `1`.
+""",
+
+        '&>>': """\
+  /// Returns the result of shifting this value's binary representation the
+  /// specified number of digits to the right.
+  ///
+  /// In a *masking shift*, the value passed as `rhs` is masked to produce a
+  /// value in the range `0..<x.bitWidth`. The shift is performed using this
```

Comment:
That's an outdated diff, which GitHub appears to no longer be flagging as such. 😕

The revised paragraph is:
```
  /// In a *masking shift*, the bit pattern of the value passed as `rhs` is
  /// masked to produce a value between zero and the bit width of `lhs`. The
  /// shift is performed using this masked value. Masking shifts require more
  /// care to use correctly than a traditional bit shift, but are likely to be
  /// more efficient when used with shift amounts that are not compile-time
  /// constants. On most architectures, a masking shift compiles down to a
  /// single instruction.
```